Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.xaml

```diff
@@ -0,0 +1,281 @@
+                        </Grid.RowDefinitions>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition x:Name="LeftColumn"
+                                              Width="Auto" />
+                            <ColumnDefinition x:Name="RightColumn"
+                                              Width="Auto" />
+                        </Grid.ColumnDefinitions>
+
+
+                        <!--Header Area-->
+                        <Grid x:Name="HeaderRoot"
+                              Grid.ColumnSpan="2"
+                              Grid.Row="0">
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" />
+                            </Grid.RowDefinitions>
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="Auto" />
+                            </Grid.ColumnDefinitions>
+
+                            <ContentPresenter x:Name="HeaderContentPresenter"
+                                              x:DeferLoadStrategy="Lazy"
+                                              VerticalAlignment="Center"
+                                              HorizontalAlignment="Left"
+                                              Grid.Row="1"
+                                              Foreground="{TemplateBinding Foreground}"
+                                              Margin="8,0"
+                                              Content="{TemplateBinding Header}"
+                                              ContentTemplate="{TemplateBinding HeaderTemplate}"
+                                              FontWeight="Normal" />
+
+                            <Button x:Name="ExpanderButton"
+                                    BorderThickness="0"
+                                    Grid.Row="1"
+                                    Foreground="{TemplateBinding Foreground}"
+                                    Grid.Column="1"
+                                    HorizontalAlignment="Left"
+                                    Background="Transparent"
+                                    VerticalAlignment="Center">
+                                <FontIcon FontFamily="Segoe MDL2 Assets"
+                                          Margin="8"
+                                          Glyph="&#xE011;" />
+                            </Button>
+                        </Grid>
+
+
+                        <!--Content Area-->
+                        <Grid x:Name="ContentRoot"
+                              Grid.ColumnSpan="2"
+                              Visibility="Collapsed"
+                              Grid.Row="1">
+
+                            <Border Child="{TemplateBinding Content}" />
+                        </Grid>
```

Comment:
Is there a need for the grid, considering there's only one child? Also there's no purpose to using a Border. Perhaps just ContentControl?